pkg/client/unversioned/request.go

```diff
@@ -641,6 +660,12 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 // fn at most once. It will return an error if a problem occurred prior to connecting to the
 // server - the provided function is responsible for handling server errors.
 func (r *Request) request(fn func(*http.Request, *http.Response)) error {
+	//Metrics for total request latency
```

Comment:
- I created a separate issue below.  I think your onto something: The semantics of that metrics are ill-defined.
- As this stands, this code doesn't change the metric.